django_project/core/settings/base.py

```diff
@@ -52,6 +52,7 @@
 # Absolute filesystem path to the directory that will hold user-uploaded files.
 # Example: "/var/www/example.com/media/"
 MEDIA_ROOT = '/home/web/django_project/media'
+MEDIA_FOLDER_PATH = '/home/web/media'
```

Comment:
hi @ann26 
i see this file is not exposed to user
could you remove this and use this folder?
https://github.com/ann26/healthsites/blob/36d2a9b20c0eb6c6a9832fae79fca36aed479659/django_project/core/settings/project.py#L124

so we have nice centered cache?